spring-content-rest/src/main/java/internal/org/springframework/content/rest/contentservice/AssociativeStoreContentService.java

```diff
@@ -344,4 +349,14 @@ public static boolean isClientAbortException(Exception e) {
         }
         return false;
     }
+
+    private void configureResourceForByteRangeRequest(RangeableResource resource, HttpHeaders headers) {
+        List<String> byteRanges = headers.get("Range");
+        if (byteRanges == null) {
+            byteRanges = headers.get("range");
```

Comment:
`HttpHeaders` is a case-insensitive map, so there is no need to check both capitalized and lowercase variants.

There is also the [`HttpHeaders#getFirst(String)`](https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/http/HttpHeaders.html#getFirst-java.lang.String-) method that can replace even more.

Reply:
Thanks.  Updated.